lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -3,7 +3,7 @@
     <div class="box text-center relative" os-dropzone="image.selectImageByPath($file)">
 
       <div class="center-block">
-        <svg-icon path="main.selection.getImageLogo() || '../assets/image.svg'"></svg-icon>
+        <svg-icon contents="main.selection.getImageLogo()" path="'../assets/image.svg'"></svg-icon>
```

Comment:
I'm guessing that `path=` here should be `paths=` ? And should these also be using array-notation like the tests are using?

Reply:
That's true, I've fixed the `path` into `paths` now